zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/UserTaskControllerTest.java

```diff
@@ -142,7 +142,7 @@ void shouldCompleteTaskWithAction(final String baseUrl) {
         .expectBody()
         .isEmpty();
 
-    Mockito.verify(userTaskServices).completeUserTask(1L, null, "customAction");
+    Mockito.verify(userTaskServices).completeUserTask(1L, Map.of(), "customAction");
```

Comment:
💭 It took me some time to understand why this change was needed, but it makes sense. The method `RequestMapper#getMapOrEmpty` didn't return an empty map if the `mapExtractor` parameter returned `null` itself. After changing this method, the test failed because an empty map is now passed to the user task service. This does not result in any different behavior though, because zeebe is able to deal with both `null` and `{}` for variables.

Reply:
Apologies, I should have made it clear in the comment or PR description 🙏